tasks/activate.yml

```diff
@@ -0,0 +1,36 @@
+---
+- name: Fill service definition template
+  ansible.builtin.template:
+    src: "sustained.service"
+    dest: "/etc/systemd/system/{{ app_name }}-{{ item.name }}.service"
+  loop: "{{ app_services }}"
+
+- name: Fill timer unit definition template
+  ansible.builtin.template:
+    src: "recurrent.service"
+    dest: "/etc/systemd/system/{{ app_name }}-{{ item.name }}.service"
+  loop: "{{ app_timers }}"
+
+- name: Fill timer definition template
+  ansible.builtin.template:
+    src: "recurrent.timer"
+    dest: "/etc/systemd/system/{{ app_name }}-{{ item.name }}.timer"
+  loop: "{{ app_timers }}"
+
+- name: Restart services
+  become: true
+  ansible.builtin.systemd:
+    name: "{{ app_name }}-{{ item.name }}.service"
+    daemon_reload: true
```

Comment:
You might want to trigger the `daemon_reload` in either a [handler](https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_handlers.html) on the templates above or in a separate step before the loops, because this will reload systemd once per service

Reply:
Oh, true. I believe the templates will always register a change because the first step in the install process is to delete all the existing service files, so I'll just add a separate step, it'll be simpler.